src/HaskellCI/Travis.hs

```diff
@@ -126,6 +126,9 @@ makeTravis argv Config {..} prj JobVersions {..} = do
         shForJob RangeGHCJS "HC=${HC}js"
         shForJob RangeGHCJS "WITHCOMPILER=\"--ghcjs ${WITHCOMPILER}js\""
 
+        -- Needed to work around haskell/cabal#6214
```

Comment:
Thanks for adding reference to the issue. I usually prefer clickable links (i.e. http urls) but this is fine  for now.